<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>211</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mechanical adj. X (mm)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ZSteeringSpinBox">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>102</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-5.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>103</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ZSteering  (mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="DistanceSkinLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>169</y>
     <width>76</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(22, 110, 255)</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>168</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(22, 110, 255)</string>
   </property>
   <property name="text">
    <string>Distance device
to target (mm) :</string>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="XMechanicSpinBox">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>209</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>241</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mechanical adj. Y (mm)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="YMechanicSpinBox">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>239</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QWidget" name="AcField_plot1" native="true">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>30</y>
     <width>868</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>272</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mechanical adj. Z (mm)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ZMechanicSpinBox">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>270</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="RefocusingcheckBox">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>23</y>
     <width>27</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>22</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(22, 110, 255)</string>
   </property>
   <property name="text">
    <string>Refocusing</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>135</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Z Rotation (degrees)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ZRotationSpinBox">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>134</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>332</y>
     <width>153</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. depth beyond
target (mm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="MaxDepthSpinBox">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>340</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>20.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>40.000000000000000</double>
   </property>
  </widget>
  <widget class="QWidget" name="IsppaScrollBars" native="true">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>410</y>
     <width>730</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="ShowWaterResultscheckBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1148</x>
     <y>417</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Show
water only</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="HideMarkscheckBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>422</y>
     <width>97</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide marks</string>
   </property>
  </widget>
  <widget class="QLabel" name="MultifocusLabel">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>300</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(22, 110, 255)</string>
   </property>
   <property name="text">
    <string>Tx Elements Set</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="SelTxSetDropDown">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>297</y>
     <width>102</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Total</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sector1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sector2</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="YSteeringSpinBox">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>75</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-5.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>76</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>YSteering  (mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>49</y>
     <width>143</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>XSteering  (mm)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="XSteeringSpinBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>48</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-5.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="LabelTissueRemoved">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>168</y>
     <width>104</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string>Tissue layers
will be removed!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculateAcField">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>373</y>
     <width>226</width>
     <height>48</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Calculate Fields</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculateMechAdj">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>420</y>
     <width>227</width>
     <height>48</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Calculate Mechanical Adjustments</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>XSteeringSpinBox</tabstop>
  <tabstop>YSteeringSpinBox</tabstop>
  <tabstop>ZSteeringSpinBox</tabstop>
  <tabstop>ZRotationSpinBox</tabstop>
  <tabstop>XMechanicSpinBox</tabstop>
  <tabstop>YMechanicSpinBox</tabstop>
  <tabstop>ZMechanicSpinBox</tabstop>
  <tabstop>SelTxSetDropDown</tabstop>
  <tabstop>MaxDepthSpinBox</tabstop>
  <tabstop>RefocusingcheckBox</tabstop>
  <tabstop>ShowWaterResultscheckBox</tabstop>
  <tabstop>HideMarkscheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
